plugins/citadel-paper/src/main/java/vg/civcraft/mc/citadel/listener/BlockListener.java

```diff
@@ -198,6 +199,22 @@ public void onBlockFromToEvent(BlockFromToEvent event) {
 		}
 	}
 
+	// prevent "enemy" grass spreading to reinforced dirt
+	@EventHandler(priority = EventPriority.LOW, ignoreCancelled = true)
+	public void onGrassSpread(BlockSpreadEvent event) {
+		if (event.getSource().getType() != Material.GRASS_BLOCK) return;
+
+		Reinforcement destRein = ReinforcementLogic.getReinforcementProtecting(event.getBlock());
+
```

Comment:
Nit: Remove newline, the if statement pertains directly to the variable declared above, they're tightly related so they should be tightly visually placed as well.